examples/08-averaging/01-average_across_bodies.py

```diff
@@ -4,19 +4,24 @@
 Average across bodies
 ~~~~~~~~~~~~~~~~~~~~~
 
-This example shows how to activate and deactivate the DPF option for averaging
-across bodies. When a multi-body simulation calculates ``ElementalNodal`` fields,
-like stresses or strains, you can either activate or deactivate the averaging
-of these fields across the different bodies when they share common nodes. This
-likely changes the end results that are shown after postprocessing of the simulation.
+In multibody simulations, some nodes may be shared by the bodies at their interfaces,
+but the values of the results, like stresses or strains, calculated at these nodes
+may differ between the bodies. This can cause discontinuous plots, given that a single
```

Comment:
```suggestion
but the values of the results (for example, stresses or strains) calculated at these nodes
may differ between the bodies. This can cause discontinuous plots, given that a single
```